monitor/monitor.go

```diff
@@ -129,16 +135,23 @@ func (m *Monitor) handleConnection(server net.Listener) {
 // mainEventLoop is responsible for reading events from the perf ring buffer
 // and manage the connections to the socket
 func (m *Monitor) mainEventLoop() {
-	log.Info("Starting monitoring traffic")
+	mutex.Lock() // Lock needed for eventConfig
```

Comment:
If the lock is needed for the `eventConfig` then add a mutex for that structure.